src/core/model_config.proto

```diff
@@ -764,6 +766,30 @@ message ModelParameter
   string string_value = 1;
 }
 
+//@@
+//@@.. cpp:var:: message ModelWarmUp
+//@@
+//@@   Settings for warming up the model instances.
+//@@
+message ModelWarmUp
+{
+  //@@  .. cpp:var:: InferRequestHeader request_header
+  //@@
+  //@@     The header message of the request sample.
+  //@@
+  InferRequestHeader request_header = 1;
+
+  //@@  .. cpp:var:: bool use_zero_value
```

Comment:
We should have a oneof with a message for each of the initialization. Like here: https://github.com/NVIDIA/tensorrt-inference-server/blob/master/src/core/grpc_service.proto#L292.

And I would start with just zero and random. Actual data case can be done but we should discuss.